spec/broadside/deploy/ecs_deploy_spec.rb

```diff
@@ -68,11 +68,59 @@
       expect { deploy.bootstrap }.to raise_error(/Service doesn't exist and no :service_config/)
     end
 
-    it 'succeeds' do
-      deploy.deploy_config.service_config = service_config
-      deploy.deploy_config.task_definition_config = task_definition_config
+    context 'with an existing task definition and service' do
+      let(:arn) { 'arn:aws:ecs:us-east-1:1234' }
```

Comment:
though it seems like you could refactor this up the rspec chain a little bit... this and a few other things (i didn't go through them all but obviously a lot of this is copy/pasted from the `deploy` section)

i personally don't really care much about maintaining spec code at a high level of awesomeness but there might be a few really easy wins here to decrease the copypasta.